src/pt/randomscan/src/eu/kanade/tachiyomi/extension/pt/randomscan/MangaDTO.kt

```diff
@@ -1,65 +1,65 @@
-package eu.kanade.tachiyomi.extension.pt.randomscan.dto
+package eu.kanade.tachiyomi.extension.pt.randomscan
 
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class Genero(
+data class GeneroDTO(
```

Comment:
`data` is not needed
```suggestion
class GeneroDTO(
```

Reply:
I've already updated it in my code. Thanks.